README.md

```diff
@@ -838,7 +839,7 @@ scrcpy -K  # short version
 
 If it fails for some reason (for example because the device is not connected via
 USB), it automatically fallbacks to the default mode (with a log in the
-console). This allows to use the same command line options when connected over
+console). This allows you to use the same command line options when connected over
```

Comment:
Couldn't we use avoid +ing here?

> This allows using the same command line…

Reply:
Yep, "This allows using the" works well. I'll make that change.